README.md

```diff
@@ -132,6 +132,7 @@ Community Maintained Cars and Features
 
 | Make      | Model (US Market Reference)   | Supported Package | ACC              | No ACC accel below | No ALC below |
 | ----------| ------------------------------| ------------------| -----------------| -------------------| -------------|
+| Audi      | A3 2015-20                    | Driver Assistance | Stock            | 0mph               | 0mph         |
```

Comment:
let’s only put the MY year of your car unless it’s been tested on all these years

Reply:
done